services/user/Supervisor/ui/src/oauth/main.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
```

Comment:
Is `@tanstack/react-query` being used anywhere in this app? Or can it be removed?